### PR TITLE
fx: static load to avoid register conflict

### PR DESCRIPTION
**Pull request by Doozers**

### CODE REVIEW

File: pgeth/engine.go [hunk 1]
```diff
@@ -59,30 +53,19 @@ func NewEngine(cfg *PluginEngineConfig) *PluginEngine {
...
-
-	for _, file := range files {
-		if file.IsDir() {
-			newPlugin, err := p.loadPluginDetailsFromDirectory(filepath.Join(pluginDirectory, file.Name()))
-			if err != nil {
-				return err
-			}
-			plugins = append(plugins, newPlugin)
+	// load monitoring plugin
+	if pluginMonitoringDirectory := os.Getenv("PGETH_MONITORING_CONFIG"); len(pluginMonitoringDirectory) != 0 {
```
Comment (mortimr) [inline pgeth/engine.go]: can we use `pluginConfigFile` like in `Start`

File: plugins/pgeth-monitoring/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+plugin.so
```
Comment (mortimr) [inline plugins/pgeth-monitoring/.gitignore]: we can remove this file

**Review by mortimr — CHANGES_REQUESTED** [review]
nits

File: pgeth/engine.go (hunk 1 above)
Comment (Doozers) [reply pgeth/engine.go]: mb

File: .gitignore [hunk 3]
```diff
@@ -53,4 +53,3 @@ tests/spec-tests/
 #pgeth
 
 !plugin/.gitkeep
```
Comment (mortimr) [inline .gitignore]: We can remove the `.gitkeep` file and remove this line

**Review by mortimr — CHANGES_REQUESTED** [review]
last nit

**Review by mortimr — APPROVED** [review]

**Pull request merged**